Mathlib/Topology/PathConnected.lean

```diff
@@ -321,24 +347,36 @@ theorem trans_apply (γ : Path x y) (γ' : Path y z) (t : I) :
 @[simp]
 theorem trans_symm (γ : Path x y) (γ' : Path y z) : (γ.trans γ').symm = γ'.symm.trans γ.symm := by
   ext t
-  simp only [trans_apply, ← one_div, symm_apply, not_le, comp_app]
-  split_ifs with h h₁ h₂ h₃ h₄ <;> rw [coe_symm_eq] at h
-  · have ht : (t : ℝ) = 1 / 2 := by linarith [unitInterval.nonneg t, unitInterval.le_one t]
-    norm_num [ht]
+  simp only [trans_apply, ← one_div, symm_apply, not_le, Function.comp_apply]
+  split_ifs with h h₁ h₂ <;> rw [coe_symm_eq] at h
+  · have ht : (t : ℝ) = 1 / 2 := by
+      refine le_antisymm h₁ ?_
+      rw [sub_le_comm] at h
+      norm_num at h
+      exact h
+    -- porting note: was `linarith [unitInterval.nonneg t, unitInterval.le_one t]` but `linarith`
+    -- doesn't know about `ℚ` yet.
```

Comment:
Is there already an issue for this problem? If so, please link to it. If not, please create it and link to it.

Reply:
It's #2714. I've now added links to it throughout.